frontend/src/app/shared/components/attachments/attachment-list/attachment-list-item.component.ts

```diff
@@ -174,14 +178,21 @@ export class AttachmentListItemComponent extends UntilDestroyedMixin implements
     return AttachmentListItemComponent.imageFileExtensions.indexOf(ext.toLowerCase()) > -1;
   }
 
-  public confirmRemoveAttachment($event:JQuery.TriggeredEvent):boolean {
-    if (!window.confirm(this.text.destroyConfirmation)) {
-      $event.stopImmediatePropagation();
-      $event.preventDefault();
-      return false;
-    }
-
-    this.removeAttachment.emit();
-    return false;
+  public confirmRemoveAttachment():void {
+    const options:ConfirmDialogOptions = {
+      text: {
+        text: this.text.deleteConfirmation,
+        title: this.text.deleteTitle,
+        button_continue: this.text.deleteTitle,
+      },
+      icon: {
+        continue: 'delete',
+      },
+      dangerHighlighting: true,
+    };
+    void this.confirmDialogService
+      .confirm(options)
+      .then(() => { this.removeAttachment.emit(); })
+      .catch(() => { /* confirmation rejected */ });
```

Comment:
You can remove the `catch` if you're not planning on doing anything with it. I think it's cleaner compared to an empty block

Reply:
Well, you can, but then you get an unhandled exception in your browser console everytime you cancel a confirmation dialog.